assets/src/edit-story/elements/gif/output.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import StoryPropTypes from '../../types';
+import MediaOutput from '../media/output';
+
+function defaultForUndefined(value, def) {
+  return value === undefined ? def : value;
+}
+
+function GifOutput({ element, box }) {
+  const { resource } = element;
+
+  const sourceProps = {
+    type: resource.output.mimeType,
+    src: resource.output.src,
+  };
+
+  const props = {
+    autoPlay: 'autoplay',
+    poster: '',
+    title: defaultForUndefined(element.title, resource.title),
+    alt: defaultForUndefined(element.alt, resource.alt),
+    layout: 'fill',
+    loop: 'loop',
+    noaudio: 'noaudio',
+  };
+
+  // crossorigin='anonymous' is required to play videos from other domains.
```

Comment:
Unrelated/Incorrect comment? `crossorigin` is nowhere set. Let's remove this comment in that case

Reply:
[Sounds like](https://amp.dev/documentation/components/amp-video/#crossorigin) we need it though?

Reply:
fyi setting crossorigin="anonymous" for video results in CORS error and no video rendering e.g.

`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://c.tenor.com/uJ7iQaKxgokAAAPo/jewelpet-ladyjewelpet.mp4. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing`

Reply:
> Sounds like we need it though?

That references `track` resources though?